modules/oc-compliance-fetching-compliance-remediation-details.adoc

```diff
@@ -0,0 +1,99 @@
+
+.Procedure
+
+. View the remediations for a profile:
++
+[source,terminal]
+----
+$ oc compliance fetch-fixes profile ocp4-cis -o /tmp
+----
++
+.Example output
+[source,terminal]
+----
+No fixes to persist for rule 'ocp4-api-server-api-priority-flowschema-catch-all' <1>
+No fixes to persist for rule 'ocp4-api-server-api-priority-gate-enabled'
+No fixes to persist for rule 'ocp4-api-server-audit-log-maxbackup'
+Persisted rule fix to /tmp/ocp4-api-server-audit-log-maxsize.yaml
+No fixes to persist for rule 'ocp4-api-server-audit-log-path'
+No fixes to persist for rule 'ocp4-api-server-auth-mode-no-aa'
+No fixes to persist for rule 'ocp4-api-server-auth-mode-node'
+No fixes to persist for rule 'ocp4-api-server-auth-mode-rbac'
+No fixes to persist for rule 'ocp4-api-server-basic-auth'
+No fixes to persist for rule 'ocp4-api-server-bind-address'
+No fixes to persist for rule 'ocp4-api-server-client-ca'
+Persisted rule fix to /tmp/ocp4-api-server-encryption-provider-cipher.yaml
+Persisted rule fix to /tmp/ocp4-api-server-encryption-provider-config.yaml
+----
+<1> The `No fixes to persist` warning is expected whenever there are rules in a profile that do not have a corresponding remediation, because either the rule cannot be remediated automatically or a remediation was not provided.
+
+. You can view a sample of the YAML file. The `head` command will show you the first 10 lines:
++
+[source,terminal]
+----
+$ head /tmp/ocp4-api-server-audit-log-maxsize.yaml
+----
++
+.Example output
+[source,terminal]
+----
+apiVersion: config.openshift.io/v1
+kind: APIServer
+metadata:
+  name: cluster
+spec:
+  maximumFileSizeMegabytes: 100
+----
+
+. View the remediation from a `ComplianceRemediation` object created after a scan:
++
+[source,terminal]
+----
+$ oc get complianceremediations
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                             STATE
+ocp4-cis-api-server-encryption-provider-cipher   NotApplied
+ocp4-cis-api-server-encryption-provider-config   NotApplied
+----
++
+[source,terminal]
+----
+$ oc compliance fetch-fixes complianceremediations ocp4-cis-api-server-encryption-provider-cipher -o /tmp
+----
++
+.Example output
+[source,terminal]
+----
+Persisted compliance remediation fix to /tmp/ocp4-cis-api-server-encryption-provider-cipher.yaml
+----
+
+. You can view a sample of the YAML file. The `head` command will show you the first 10 lines:
++
+[source,terminal]
+----
+$ head /tmp/ocp4-cis-api-server-encryption-provider-cipher.yaml
+----
++
+.Example output
+[source,terminal]
+----
+apiVersion: config.openshift.io/v1
+kind: APIServer
+metadata:
+  name: cluster
+spec:
+  encryption:
+    type: aescbc
+----
```

Comment:
You might want to add a warning in the end about the remediations. That special care needs to be taken before applying them directly, since some might not be applicable in bulk (e.g. the usbguard rules in the moderate profile). In such cases, it is preferrable to let the Compliance Operator apply the rules as it takes care of the dependencies and ensures that the cluster remains in a good state.